nova_vm/src/engine/bytecode/executable.rs

```diff
@@ -284,7 +284,7 @@ impl Executable {
             .push(unsafe { std::mem::transmute::<Instruction, u8>(instruction) });
     }
 
-    fn add_instruction(&mut self, instruction: Instruction) {
+    pub(super) fn add_instruction(&mut self, instruction: Instruction) {
```

Comment:
**nitpick:** You don't need `pub(super)` for a submodule of the current module. That's how `for_in_of_statement` and `function_declaration_instantiation` are adding instructions.

Reply:
Oops <3